rgw/v2/tests/aws/reusable.py

```diff
@@ -75,6 +75,132 @@ def list_object_versions(bucket_name, end_point, ssl=None):
         raise AWSCommandExecError(message=str(e))
 
 
+def create_multipart_upload(bucket_name, key_name, end_point, ssl=None):
+    """
+    Initiate multipart uploads for given object on a given bucket
+    Ex: /usr/local/bin/aws s3api create-multipart-upload --bucket <bucket_name> --key <key_name> --endpoint <endpoint_url>
+    Args:
+        bucket_name(str): Name of the bucket
+        key_name(str): Name of the object for which multipart upload has to be initiated
+        end_point(str): endpoint
+        ssl:
+    Return:
+        Response of create-multipart-upload
+    """
+    list_method = AWS(operation="create-multipart-upload")
```

Comment:
@ckulal could you use the proper variable name like aws_conn in all methods 
list_method is misleading

Reply:
done